docs/im.md

```diff
@@ -0,0 +1,27 @@
+### IM API
+
+RocketChat's Im Messages.
+
+#### im.create
+
+Create a direct message.
+
+```ruby
+require 'rocketchat'
+
+rocket_server = RocketChat::Server.new('http://your.server.address/')
+session = rocket_server.login('username', 'password')
+im = session.im.create(username: 'rocket.cat')
```

Comment:
it looks like the `create` endpoint returns a `room` and not an `im`

Reply:
Update too.